Common subdirectories: xam/cli and xam.new/cli
diff -c xam/repos.py xam.new/repos.py
*** xam/repos.py	2015-07-17 11:43:31.503793533 +0200
--- xam.new/repos.py	2015-07-17 11:39:39.351793878 +0200
***************
*** 12,17 ****
--- 12,34 ----
      :license: BSD, see LICENSE for more details.
  '''
  
+ ISENGARD= (
+     'http://mirrors.xbmc.org/addons/isengard/addons.xml',
+     'http://mirrors.xbmc.org/addons/isengard/',
+     'http://mirrors.xbmc.org/addons/isengard/addons.xml.md5',
+ )
+ 
+ HELIX= (
+     'http://mirrors.xbmc.org/addons/helix/addons.xml',
+     'http://mirrors.xbmc.org/addons/helix/',
+     'http://mirrors.xbmc.org/addons/helix/addons.xml.md5',
+ )
+ 
+ GOTHAM= (
+     'http://mirrors.xbmc.org/addons/gotham/addons.xml',
+     'http://mirrors.xbmc.org/addons/gotham/',
+     'http://mirrors.xbmc.org/addons/gotham/addons.xml.md5',
+ )
  
  FRODO= (
      'http://mirrors.xbmc.org/addons/frodo/addons.xml',
***************
*** 19,50 ****
      'http://mirrors.xbmc.org/addons/frodo/addons.xml.md5',
  )
  
- FRODO_PRE = (
-     'http://mirrors.xbmc.org/addons/frodo-pre/addons.xml',
-     'http://mirrors.xbmc.org/addons/frodo-pre/',
-     'http://mirrors.xbmc.org/addons/frodo-pre/addons.xml.md5',
- )
- 
- DHARMA = (
-     'http://mirrors.xbmc.org/addons/dharma/addons.xml',
-     'http://mirrors.xbmc.org/addons/dharma/',
-     'http://mirrors.xbmc.org/addons/dharma/addons.xml.md5',
- )
- 
- DHARMA_PRE = (
-     'http://mirrors.xbmc.org/addons/dharma-pre/addons.xml',
-     'http://mirrors.xbmc.org/addons/dharma-pre/',
-     'http://mirrors.xbmc.org/addons/dharma-pre/addons.xml.md5',
- )
- 
  EDEN = (
      'http://mirrors.xbmc.org/addons/eden/addons.xml',
      'http://mirrors.xbmc.org/addons/eden/',
      'http://mirrors.xbmc.org/addons/eden/addons.xml.md5',
  )
  
! EDEN_PRE = (
!     'http://mirrors.xbmc.org/addons/eden-pre/addons.xml',
!     'http://mirrors.xbmc.org/addons/eden-pre/',
!     'http://mirrors.xbmc.org/addons/eden-pre/addons.xml.md5',
  )
--- 36,49 ----
      'http://mirrors.xbmc.org/addons/frodo/addons.xml.md5',
  )
  
  EDEN = (
      'http://mirrors.xbmc.org/addons/eden/addons.xml',
      'http://mirrors.xbmc.org/addons/eden/',
      'http://mirrors.xbmc.org/addons/eden/addons.xml.md5',
  )
  
! DHARMA = (
!     'http://mirrors.xbmc.org/addons/dharma/addons.xml',
!     'http://mirrors.xbmc.org/addons/dharma/',
!     'http://mirrors.xbmc.org/addons/dharma/addons.xml.md5',
  )
